1-mandatory/2-quotegenerator/quotes.js

```diff
@@ -17,8 +17,20 @@
 // pickFromArray(coloursArray)  //maybe returns "#F38630"
 //
 // You DO NOT need to understand how this function works.
-function pickFromArray(choices) {
-  return choices[Math.floor(Math.random() * choices.length)];
+// function pickFromArray(choices) {
+//   return choices[Math.floor(Math.random() * choices.length)];
+// }
+
+
+const button = document.getElementById("button");
+
+button.addEventListener("click", function () {
+  newQuote();
+});
```

Comment:
this is the exact same as writing `button.addEventListener("click", newQuote)`; writing a function to wrap the execution of another function is redundant here